tests/ch9329.test.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 Khalil Estell
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <libhal-input/ch9329.hpp>
+
+#include <boost/ut.hpp>
+
+namespace hal::input {
+void ch9329_test()
+{
+  using namespace boost::ut;
+  using namespace std::literals;
+
+  "ch9329::create()"_test = []() {
```

Comment:
```suggestion
  "ch9329::ch9329()"_test = []() {
```

Since we don't use factory functions anymore.